src/handlers/versionManager.ts

```diff
@@ -0,0 +1,8 @@
+import { version_2_6_1 } from '../../template/2.6.1/version_2_6_1';
+import { version_3_0_0 } from '../../template/3.0.0/version_3_0_0';
+
+const availableVersions = [version_2_6_1, version_3_0_0];
+
+export const getVersionConfig = (name: string) => {
+  return availableVersions.find(version => version.isSatisfiedBy(name));
```

Comment:
You should return full version config here (so actually call create on found version). What's more it is not protected from wrong version... there should be error being thrown.